Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -123,6 +123,12 @@ public async Task<PackageInfo[]> GetPackages(string manifestName, string manifes
                             continue;
                         }
 
+                        // Only show plugins that are less than or equal to maximumAbi.
+                        if (Version.TryParse(ver.MaximumAbi, out var maximumAbi) && _applicationHost.ApplicationVersion <= maximumAbi)
```

Comment:
With this change, if the `maximumAbi` matches, the `targetAbi` isn't checked anymore. The logic of this whole loop should probably be inverted to fix it, and handle both Abi fields properly.

Reply:
Don't you mean to swap the order of the checks?

As it stands, you can install a plugin that is made for an older server version. You can also install a version meant for a newer server version than you are running. The problem is that when a plugin updates to the higher version before the server, the user is left in an awkward situation where they have to prevent updates. By placing a cap on the version, you prevent that from happening. Setting a max ABI acts as that cap, so you want the server version to be **lower** than the cap.

Reply:
I think i tested all use cases but i'll do my tests again

Reply:
the continue skips the `if (_applicationHost.ApplicationVersion >= targetAbi)` check?